masterlist.yaml

```diff
@@ -5694,15 +5357,7 @@ plugins:
       - crc: 0xB337055B # Modular - Core v1.81
         util: 'SSEEdit v3.2.84 EXPERIMENTAL'
     msg:
-      - <<: *hasWildEdits
-        subs:
-          - |+
-
-              - Open SSEEdit and load **Distinct Interiors.esp**.
```

Comment:
Same as above, removing this loses information.

Reply:
https://github.com/loot/skyrimse/commit/2acab29df606cad6755c3d5eb565de684ee322d5